README.md

```diff
@@ -8,7 +8,7 @@ for use by [Horizontal Pod Autoscaler] and [Vertical Pod Autoscaler]. Metrics AP
 making it easier to debug autoscaling pipelines.
 
 > [!CAUTION]
-> Metrics Server is not meant for non-autoscaling purposes. For example, don't use it to forward metrics to monitoring solutions, or as a source of monitoring solution metrics. In such cases please collect metrics from Kubelet `/metrics/resource` endpoint directly.
+> Metrics Server is meant for autoscaling purposes. For example, don't use it to forward metrics to monitoring solutions, or as a source of monitoring solution metrics. In such cases please collect metrics from Kubelet `/metrics/resource` endpoint directly.
```

Comment:
```suggestion
> Metrics Server is meant only for autoscaling purposes. For example, don't use it to forward metrics to monitoring solutions, or as a source of monitoring solution metrics. In such cases please collect metrics from Kubelet `/metrics/resource` endpoint directly.
```

Reply:
I felt like adding that also, thought it might have been over stepping the meaning.